Gemfile

```diff
@@ -36,9 +36,6 @@ gem 'zendesk_api'
 gem 'cf-app-utils'
 gem 'health_check', '~> 3.0'
 
-# Email and Text Notifications
-gem 'govuk_notify_rails'
```

Comment:
Notify was used solely for sending admin emails like password resets as far as I can see.